src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -1972,6 +1992,7 @@ abstract class DatabaseTest {
     val plantingSubzoneIds = mutableListOf<PlantingSubzoneId>()
     val plantingZoneIds = mutableListOf<PlantingZoneId>()
     val projectIds = mutableListOf<ProjectId>()
+    val projectDocumentSettingsProjectIds = mutableListOf<ProjectId>()
```

Comment:
Maybe this isn't needed? The IDs here ought to always be in `projectIds`.

Reply:
Good call